vaadin-maps-leaflet-flow/pom.xml

```diff
@@ -231,6 +231,14 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
```

Comment:
The component is packaged as a jar. There is no need to add the maven-war-plugin. Please delete this change.